functions/nodejs-apps.bash

```diff
@@ -214,9 +214,8 @@ nodered_setup() {
 zigbee2mqtt_download() {
   echo -n "$(timestamp) [openHABian] Downloading Zigbee2MQTT... "
   if ! cond_redirect mkdir -p /opt/zigbee2mqtt; then echo "FAILED (mkdir -p /opt/zigbee2mqtt)"; fi
-  if ! cond_redirect chown openhabian /opt/zigbee2mqtt; then echo "FAILED (chown /opt/zigbee2mqtt)"; fi
-  if ! cond_redirect chgrp openhab /opt/zigbee2mqtt; then echo "FAILED (chgrp /opt/zigbee2mqtt)"; fi
-  if ! cond_redirect sudo -u "${username:-openhabian}" git clone https://github.com/Koenkk/zigbee2mqtt.git "/opt/zigbee2mqtt"; then echo "FAILED (git clone)"; return 1; fi
+  if ! cond_redirect chown "zigbee2mqtt:${username:-openhabian}" /opt/zigbee2mqtt; then echo "FAILED (chown /opt/zigbee2mqtt)"; fi
```

Comment:
owner=zigbee2mqtt but group=ohian-username ?
Do you really want to introduce a new Linux user ?
Why?

And the group should be "openhab", not the username

Reply:
yes, a seperate user is imo cleaner as zigbee2mqtt runs it's own service and the connection to openhab is just mqtt. And then it's no problem if the openhab user is changed.
I was not sure which group I should use. So I change to openhab, no problem

Reply:
You should be avoiding a new user if possible, lots of potential things to go wrong on OHian updates.
Access rights on serial device, too.

Plus, also think of the uninstall scenario.

Pls re-think if you can run it as the openhabian Linux user.

Reply:
no problem: I switched back to $openhabian. Just thought a seperate user is cleaner